python/ray/dashboard/client/src/pages/actor/ActorDetail.tsx

```diff
@@ -133,6 +133,15 @@ const ActorDetailPage = () => {
                 }
               : { value: "-" },
           },
+          {
+            label: "Actor PID",
```

Comment:
```suggestion
            label: "PID",
```

To align with what we have in some other places.